src/util.h

```diff
@@ -562,6 +562,10 @@ class BufferValue : public MaybeStackBuffer<char> {
   inline std::string_view ToStringView() const {
     return std::string_view(out(), length());
   }
+  inline std::u8string_view ToStringU8View() const {
```

Comment:
I have some doubts about the name of the `ToStringU8View` method.
Since the method of returning std::string_view is called ToStringView, why is the method of returning std::u8string_view not `ToU8StringView` but `ToStringU8View`

Reply:
Yeah, after reading this, I agree.